pyproject.toml

```diff
@@ -2,3 +2,4 @@
 target-version = ['py36', 'py37']
 skip-string-normalization = true
 line-length = 79
+exclude = 'vendored\|_vispy'
```

Comment:
test is failing because this somehow did not properly exclude the desired directories

Reply:
yeah same command line string not working